distribution/lib/Standard/Base/0.0.0-dev/src/Logging.enso

```diff
@@ -0,0 +1,37 @@
+import project.Any.Any
+import project.Data.Text.Text
+import project.Meta
+import project.Nothing.Nothing
+
+polyglot java import java.util.logging.Logger
+
+## Log a message.
+   This function needs to be enabled by importing `Standard.Base.Logging` using
+   `from Standard.Base.Logging import all`.
+Any.log : Text -> Log_Level -> Any
```

Comment:
Shouldn't `Any.log` be private? I agree with Jaroslav that there is no way to configure it atm so lets keep it simple.